ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
The tinyMCE document said that if we don't use `p` as root elements, we may cripple its functionalities. I don't know why but I suppose that tinyMCE use the `p` tag as "marker" to know stuffs (where to insert new text, where to display or not the insert toolbar, ...).

As tinyMCE is configured to add and use a  `p` everywhere in the generated html (and so in the input html if the user what to modify it), I don't want to store in the db and give to tinyMCE "wrong" html input.